CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ All notable changes to the Pony compiler and standard library will be documented
 - Logger package
 - `ArrayValues.rewind()` method.
 - Nanos primitive in time package.
+- An initial implementation of WriteBuffer inside the net package
```

Comment:
This seems more like a commit message than a changelog message - I would drop the part about being an initial implementation - as this tidbit is not really relevant to the changelog.